<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translations</class>
 <widget class="QDialog" name="Translations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map translations</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="languageSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="supportedCheck">
       <property name="text">
        <string>Supported</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="translationsTable">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>240</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
     <column>
      <property name="text">
       <string>String ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Text</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
